_vector-search/ai-search/agentic-search.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: default
+title: Agentic search
+parent: AI search
+nav_order: 30
+has_children: false
+---
+
+# Agentic Search
+Introduced 3.2
+{: .label .label-purple }
+
+Agentic search lets users ask questions in natural language and have OpenSearch plan and execute the retrieval automatically. A preconfigured **agent** reads the question, plans the search, and returns relevant results.
+
+**PREREQUISITE**<br>
+Before using using agentic search, you must configure an agent with QueryPlanningTool. For more information, see..
```

Comment:
Need to add page link of https://github.com/opensearch-project/documentation-website/pull/10785 once the PR is merged